site/templates/film/item.jet

```diff
@@ -70,28 +70,27 @@
             {{end}}
             <div class="{{ isEnabled("element_switcher_enabled") ? "s72-hide" : "" }}"> {* hide class prevents FOUC *}
               <div class="meta-detail-synopsis">{{film.Overview | raw}}</div>
-              <div class="meta-detail-cast">
-              {{if len(film.Cast) > 0 }}
-                <h2>{{i18n("meta_detail_cast_title")}}</h2>
-              {{end}}
-                <p>
-                  {{range index, member := film.Cast}}
-                    {{ path := "/search.html?q=" + member.Name }}
-                    <a href="{{ routeToPath(path) }}">{{member.Name}}</a>{{if index < len(film.Cast) - 1}}, {{end}}
-                  {{end}}
-                </p>
-              </div>
-              <div class="meta-detail-crew d-flex">
+                <div class="mb-2 row">
```

Comment:
Don't think that div needs to be indented there!

Reply:
Ah I see that div just before it closes on the same line. Thanks - have unindented.